app/models/support/requests/content_publisher_feedback_request.rb

```diff
@@ -0,0 +1,30 @@
+require 'active_support/core_ext'
+
+module Support
+  module Requests
+    class ContentPublisherFeedbackRequest < Request
+      attr_accessor :feedback_type, :feedback_details, :impact_on_work
+
+      validates_presence_of :feedback_type, :feedback_details
+      validates :feedback_type, inclusion: { in: ["accessibility or usability", "working unexpectedly", "helpful feature", "other"] }
```

Comment:
Rather than repeating these can we have a single definition to check them? - Maybe have feedback_type_options as a constant?